server/controller/auth/registerController.js

```diff
@@ -0,0 +1,78 @@
+var CryptoJS = require("crypto-js");
```

Comment:
```suggestion
const CryptoJS = require("crypto-js");
```
Will there be no problem, if it is left as var?